_posts/2020-04-28-changes-to-cloud-gov’s-existing-cdn-and-custom-domain-services-1.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title: Changes to cloud.gov’s existing CDN and custom domain services
+date: April 28, 2020
+---
+This message is to let you know about upcoming changes to cloud.gov’s CDN and domain services.
+
+**What is happening to the old services?**
+
+Let’s Encrypt, which the cloud.gov custom domain service uses to provision SSL certificates for custom domains, is deprecating their v1 API. If you already have a domain registered, you will not be impacted, but you should still prepare for changes.
+
+**Here are the details:**
+
+cloud.gov’s existing CDN service (provided by the cdn-broker) and the custom-domain service (provided by the domains-broker) will no longer be able to provision *new domains* starting June 1, 2020. All *existing domains* will continue to operate normally.
+
+We are working on a new version of the service that will use the Let’s Encrypt v2 API and anticipate having a new service in place before the existing services retire in June.
+
+We’ve attempted to answer some questions below, but as always, feel free to reach out to us directly at support@cloud.gov.
+
+**What is a service broker?**
+
+Service brokers are a fundamental part of the cloud.gov platform. They allow developers to provision production-grade third-party services though the familiar Cloud Foundry interface. Some brokers, like the custom domain broker, orchestrate multiple services and configure the complex interactions. This saves countless hours of developer time and allows the platform to manage these services on your behalf.
+
+**What are the cdn and domains services used for?**
+
+You can see the cdn and domains services in the [Dashboard](https://dashboard.fr.cloud.gov/), or by running cf marketplace from the command line:
+
+`$ cf marketplace`
```

Comment:
Can you use the tripple-backticks instead of single here?  Like this:

```
$ cf marketplace
cdn-route ...
custom-domain ...
```

![Create Posts “2020-04-28-changes-to-cloud-gov’s-existing-cdn-and-custom-domain-services-1” by karareinsel · Pull Request #1653 … 2020-04-28 11-09-42](https://user-images.githubusercontent.com/2016/80522026-c4bf3900-8940-11ea-8873-0dbbb15c3922.png)